janome/dic.py

```diff
@@ -396,7 +396,7 @@ class UserDictionary(RAMDictionary):
     User dictionary class (on-the-fly)
     """
 
-    def __init__(self, user_dict, enc, type, connections):
+    def __init__(self, user_dict, enc, type, connections, progress_handler=None):
```

Comment:
Please add description to the docstring for the newly added parameter.

Reply:
Added! 👉 https://github.com/mocobeta/janome/pull/87/commits/08d8a766effac2054089796a92bddb31ffc29b1a